.github/workflows/ci-opensearch.yml

```diff
@@ -16,6 +16,9 @@ jobs:
         - major: 1.x
           image: 1.0.0
           distribution: opensearch
+        - major: 2.x
```

Comment:
I think we might need to add a test case for 2.x in: https://github.com/jaegertracing/jaeger/blob/main/pkg/es/client/cluster_client_test.go

Reply:
Added, please review.